lib/cn/kvset.h

```diff
@@ -348,6 +350,10 @@ kvset_pfx_lookup(
     struct kvs_buf *       vbuf,
     struct query_ctx *     qctx);
 
+/* MTF_MOCK */
+bool
+kvset_younger(const struct kvset *ks1, const struct kvset *ks2);
```

Comment:
A comment saying what true and false represent would be useful so we don't have to go read the code